src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -22,18 +25,26 @@ class AddressValidationModal extends React.Component {
       validationKey,
       addressValidationType,
       selectedAddress,
+      selectedId,
     } = this.props;
+
     const payload = {
       ...selectedAddress,
       validationKey,
-      addressPou:
-        addressValidationType === VET360.FIELD_NAMES.MAILING_ADDRESS
-          ? VET360.ADDRESS_POU.CORRESPONDENCE
-          : VET360.ADDRESS_POU.RESIDENCE,
     };
 
     const method = payload.id ? 'PUT' : 'POST';
 
+    if (selectedId !== 'userEntered') {
+      this.props.updateValidationKeyAndSave(
+        VET360.API_ROUTES.ADDRESSES,
+        method,
+        addressValidationType,
+        payload,
+        this.props.analyticsSectionName,
+      );
+    }
+
     this.props.createTransaction(
```

Comment:
I think we need to wrap this call in an `else` so we aren't making two calls if the user picks a suggestion.

Reply:
Fixed